k8s/apis/model/v1/common_types.go

```diff
@@ -50,6 +59,33 @@ type TargetSpec struct {
 	// Defines the version of a particular resource
 	Version    string `json:"version,omitempty"`
 	Generation string `json:"generation,omitempty"`
+
+	// Optional ReconcilicationPolicy to specify how target controller should reconcile.
+	// Now only periodic reconciliation is supported. If the interval is 0, it will only reconcile
+	// when the instance is created or updated.
+	ReconciliationPolicy *ReconciliationPolicySpec `json:"reconciliationPolicy,omitempty"`
+}
+
+// +kubebuilder:object:generate=true
+type InstanceSpec struct {
+	Name        string                       `json:"name"`
+	DisplayName string                       `json:"displayName,omitempty"`
+	Scope       string                       `json:"scope,omitempty"`
+	Parameters  map[string]string            `json:"parameters,omitempty"` //TODO: Do we still need this?
+	Metadata    map[string]string            `json:"metadata,omitempty"`
+	Solution    string                       `json:"solution"`
+	Target      model.TargetSelector         `json:"target,omitempty"`
+	Topologies  []model.TopologySpec         `json:"topologies,omitempty"`
+	Pipelines   []model.PipelineSpec         `json:"pipelines,omitempty"`
+	Arguments   map[string]map[string]string `json:"arguments,omitempty"`
+	Generation  string                       `json:"generation,omitempty"`
+	// Defines the version of a particular resource
+	Version string `json:"version,omitempty"`
```

Comment:
Can we create a separate issue to review this attribute? I think this was introduced by AIO team earlier, not sure what's the functionality is.

Reply:
Sure. Open the issue https://github.com/eclipse-symphony/symphony/issues/235. And I found I used to check this field with Nonso, he confirmed this field is not used and we can safely remove it.

Reply:
Removed this property.